_posts/2016-02-04-ombulabs-products.markdown

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: "New OmbuLabs Products"
```

Comment:
@cecilia-ombulabs Maybe change it and call it "Announcing The Ombu Labs Shop!" or something like that?